group_vars/all.yml

```diff
@@ -70,6 +70,16 @@ scale:
         rhel7_interfaces: [em1, em2, p1p1, p1p2]
         rhel8_interfaces: []
 
+# Scale lab can provide us public external network on 4th interface
+# We can use that to access overcloud VM's from outside undercloud.
+# To use this feature set 'public_external_interface: true' and
+# assign 4th nic to external_interface to 4th nic and
+# define external_XXXXXX parameters wth CIDR provided by lab team
+# Don't define external_network_vlan_id as lab team might have
+# created their own vlan for this external network implementation.
+#public_external_interface: false
+#external_interface: ens2f3
```

Comment:
You should make this clear, if its a rhel7 interface or rhel 8 interface.

Reply:
I have added that in README.md
https://github.com/redhat-performance/jetpack/pull/216/files#diff-04c6e90faac2675aa89e2176d2eec7d8R147

Reply:
Yes, but I don't think it mentiones anything about rhel8/rhel7 naming convention. Atleast, you might want to add that in the README?